Woof/WoofTests/Helper/DataTransformerTests/FiveStarRatingTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+final class FiveStarRatingTests: XCTestCase {
+    func testFiveStarRatingReturnsCaseUnavailableWhenNegativeValueProvided() {
+        XCTAssertEqual(DataTransformer.fiveStarRating(for: -25.0), .unavailable)
+    }
+
+    func testFiveStarRatingReturnsCaseUnavailableWhenZeroValueProvided() {
+        XCTAssertEqual(DataTransformer.fiveStarRating(for: 0), .unavailable)
+    }
+
+    func testFiveStarRatingReturnsCaseUnavailableWhenValueBetweenZeroAndOneHalfValueProvided() {
+        XCTAssertEqual(DataTransformer.fiveStarRating(for: 0.35), .unavailable)
+    }
+
+    func testFiveStarRatingReturnsOneStarWhenOneHalfValueProvided() {
+        XCTAssertEqual(DataTransformer.fiveStarRating(for: 0.5), .rated(.oneStar))
+    }
```

Comment:
nit:

Put all these case into one test:

```suggestion
    func testFiveStarRatingReturnsExpectedStarsForTheGivenValues() {
        XCTAssertEqual(DataTransformer.fiveStarRating(for: 0.5), .rated(.oneStar))
        XCTAssertEqual(DataTransformer.fiveStarRating(for: 1.0), .rated(.oneStar))
        // and so on till 5.0
    }
```

Reply:
Done.

Reply:
Sorry, fixed it.

Reply:
Why do you need the following tests:
- `testReturnsOneStarWhenValueBetweenOneHalfAndOneProvided`;
- `testReturnsFourStarsWhenValueBetweenFourAndFourAndHalfProvided`;
- `testReturnsFiveStarsWhenValueBetweenFourAndHalfAndFiveProvided`;
?

I thought the `testReturnsExpectedStarsForTheGivenValues` covers cases in the `0.1-5.0` range.
Either put mentioned tests into the `testReturnsExpectedStarsForTheGivenValues ` or remove them.

Reply:
Ok, removed.